manim/mobject/svg/tex_mobject.py

```diff
@@ -248,15 +244,15 @@ def sort_alphabetically(self):
         self.submobjects.sort(key=lambda m: m.get_tex_string())
 
 
-class TextMobject(TexMobject):
+class Tex(MathTex):
     CONFIG = {
         "alignment": "\\centering",
         "arg_separator": "",
         "type": "text",
     }
```

Comment:
Comment (non-blocking): What never made sense to me is that I would expect MathTex() to be equivalent to something like Tex(mathmode=True) or something like that. But it seems to be the other way around, and you can't even really make one from the other. But anyway.

Reply:
we should probably change this at some point